Module2/springhometask/pom.xml

```diff
@@ -34,6 +33,13 @@
             <artifactId>spring-context-support</artifactId>
             <version>${spring.version}</version>
         </dependency>
+
+        <dependency>
```

Comment:
how this dependency is used spring-jdbc?

Reply:
it is not needed for now, ok deleted